boards/x86/qemu_x86/doc/board.rst

```diff
@@ -6,9 +6,8 @@ X86 Emulation (QEMU)
 Overview
 ********
 
-The Zephyr Kernel uses the qemu_x86 board configuration to emulate Pentium-class
-systems running on QEMU.
-This board configuration provides support for an x86 Minute IA CPU and the
+The X86 QEMU board configuration is used to emulate the X86 architecture. This board
```

Comment:
Same comment.  You should mention which board configuration you're referring to:

    The `BOARD=qemu_x86` board configuration ...